src/wallet/rpcwallet.cpp

```diff
@@ -39,6 +39,10 @@
 
 static const std::string WALLET_ENDPOINT_BASE = "/wallet/";
 
+static const uint32_t WALLET_MIN_FEERATE = 100; // sat/b
```

Comment:
This is wrong. Relay min feerate is already defined elsewhere, and user-configurable.

Reply:
Using `DEFAULT_MIN_RELAY_TX_FEE` instead.

Reply:
Actually, I would have to include `validation.h` to access that, and I can't access it via `extern` because it's `static`. I'm just doing `1000 /* DEFAULT_MIN_RELAY_TX_FEE */` for now.

Reply:
Using the default would be wrong too. It should use the configured policy or current mempool limit...

Reply:
I don't see a problem with someone making a tx that is not relayable at this point in time (but relayable when tx traffic drops). In fact, I'm not even sure this check is necessary in the first place.

Reply:
In that case, it should be the *configured* relay minimum policy (or no check at all).

Reply:
I'm honestly leaning towards removing this check. Thanks for the nudge.